services/docker-compose.yaml

```diff
@@ -5,7 +5,7 @@ services:
     container_name: idw-keria
     restart: unless-stopped
     build:
-      context: github.com/WebOfTrust/keria#f24cf4b01932916cc640ff6f20ffda6f641c1ad2
+      context: github:cardano-foundation/signify-ts#c1422b041d2ff53b3d4456a29443722b95f35fb5
```

Comment:
Actually we don't need to use our fork since this is on their main branch now. Can you check if everything still works when you use the latest on the main branch on `WebOfTrust/keria`?

If something breaks, maybe we can just step back in commits until it works. (in case Signify and KERIA are out of sync)

Reply:
`da8e53fe92b9027ec5b547c00d4f54f278fbb1b2` deployed now @bao-sotatek